src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/form/main.js

```diff
@@ -410,11 +289,19 @@ define(['config'], function(Config) {
             ]);
         },
 
+        formInitializedHandler: function(data) {
+            this.setFormData(data);
+        },
+
         setFormData: function(data) {
             // add collection filters to form
             this.sandbox.emit('sulu.contact-form.add-collectionfilters', this.form);
+
+            this.numberOfBankAccounts = !!data.bankAccounts ? data.bankAccounts.length : 0;
+            this.updateBankAccountAddIcon(this.numberOfBankAccounts);
+
             this.sandbox.form.setData(this.form, data).then(function() {
-                this.sandbox.start(this.form);
+                this.sandbox.start('#contact-fields');
```

Comment:
Is there a reason for this change? At least I would put that string into the `constants`-variable.